libp2p/peer/peerstore.py

```diff
@@ -23,65 +36,65 @@ def __create_or_get_peer(self, peer_id):
         self.peer_map[peer_id] = data
         return self.peer_map[peer_id]
 
-    def peer_info(self, peer_id):
+    def peer_info(self, peer_id: ID) -> Optional[PeerInfo]:
         if peer_id in self.peer_map:
-            peer = self.peer_map[peer_id]
-            return PeerInfo(peer_id, peer)
+            peer_data = self.peer_map[peer_id]
+            return PeerInfo(peer_id, peer_data)
         return None
 
-    def get_protocols(self, peer_id):
+    def get_protocols(self, peer_id: ID) -> List[str]:
         if peer_id in self.peer_map:
             return self.peer_map[peer_id].get_protocols()
         raise PeerStoreError("peer ID not found")
 
-    def add_protocols(self, peer_id, protocols):
+    def add_protocols(self, peer_id: ID, protocols: Sequence[str]) -> None:
         peer = self.__create_or_get_peer(peer_id)
-        peer.add_protocols(protocols)
+        peer.add_protocols(list(protocols))
 
-    def set_protocols(self, peer_id, protocols):
+    def set_protocols(self, peer_id: ID, protocols: Sequence[str]) -> None:
         peer = self.__create_or_get_peer(peer_id)
-        peer.set_protocols(protocols)
+        peer.set_protocols(list(protocols))
 
-    def peers(self):
+    def peer_ids(self) -> List[ID]:
         return list(self.peer_map.keys())
 
-    def get(self, peer_id, key):
+    def get(self, peer_id: ID, key: Any) -> Any:
         if peer_id in self.peer_map:
             val = self.peer_map[peer_id].get_metadata(key)
             return val
         raise PeerStoreError("peer ID not found")
 
-    def put(self, peer_id, key, val):
+    def put(self, peer_id: ID, key: Any, val: Any) -> None:
```

Comment:
https://github.com/libp2p/go-libp2p-core/blob/master/peerstore/peerstore.go#L80-L86
```suggestion
    def put(self, peer_id: ID, key: str, val: Any) -> None:
```